src/MixedQuoter.sol

```diff
@@ -155,11 +156,13 @@ contract MixedQuoter is IMixedQuoter, IPancakeV3SwapCallback {
         public
         view
         override
-        returns (uint256 amountOut)
+        returns (uint256 amountOut, uint256 gasEstimate)
     {
+        uint256 gasBefore = gasleft();
         (uint256 reserveIn, uint256 reserveOut) =
             V3SmartRouterHelper.getReserves(factoryV2, params.tokenIn, params.tokenOut);
         amountOut = V3SmartRouterHelper.getAmountOut(params.amountIn, reserveIn, reserveOut);
+        gasEstimate = gasBefore - gasleft();
     }
```

Comment:
Head up: this could inaccurate since actual swap is not performed. Feel free if you guys have better solution

Reply:
lets create GH issue on this and take it from there. 

1. Fixing it in this PR will expand the scope
2. its used by FE so we can eventually do a `MixedQuoterV2` with proper gas estimate for v2 and stableswap